templates/index.html

```diff
@@ -3,7 +3,10 @@
 {% block content %}
 <div class="jumbotron">
   <p>
-    {{settings.description|safe}}
+    We are a small group of friends with a passion for programming. We like experimenting with new technology and learning new things. On our days off we try to make up as many fun ideas as possible which we then try to implement. We love weekend projects. Our goal in life is to make something new or useful or both!{{settings.description|safe}}
```

Comment:
Ce tole mergam, bo na psywerx.net ta description 2x. Mogoce bi bilo bols dat tole v fixture (model je web.models.Static pa rabis samo en row, kjer dolocis title, description in footer.